Competition2017/tests/competition/subsystems/climbing/commands/IntakeAtLowPowerForTimeTest.java

```diff
@@ -0,0 +1,35 @@
+package competition.subsystems.climbing.commands;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+import org.junit.Test;
+
+import competition.BaseTest;
+import competition.subsystems.climbing.ClimbingSubsystem;
+import edu.wpi.first.wpilibj.MockTimer;
+
+public class IntakeAtLowPowerForTimeTest extends BaseTest {
+    MockTimer timer;
+    
+    @Test
+    public void executeTest(){
+        timer = injector.getInstance(MockTimer.class);
+        IntakeAtLowPowerForTimeCommand command = injector.getInstance(IntakeAtLowPowerForTimeCommand.class);
+        ClimbingSubsystem subsystem = injector.getInstance(ClimbingSubsystem.class);
+
+        command.setTimeoutForIntake(0.5);
+        command.setLowPowerForIntake(0.3);
+        command.initialize();
+        command.execute();
+        assertFalse(subsystem.brakeSolenoid.get());
+        assertTrue(subsystem.isAscending);
+        timer.advanceTimeInSecondsBy(0.5);
+        assertTrue(command.isFinished());
+        command.end();
+        assertTrue(subsystem.brakeSolenoid.get());
+        assertFalse(subsystem.isAscending);
+        
+    }
+
```

Comment:
You should consider having another test where, before runnin gthe command, you advance time by 120 seconds (e.g. near the end of a match), since that's when this climbing stuff will happen.